### ISSUE TITLE
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. Setting `pad_token_id` to `eos_token_id`:50256 for open-end generation.

### ISSUE THREAD

**Issue by nekohacker591** [KoboldAI/KoboldAI-Client#382]
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:50256 for open-end generation.

**Comment by Soumendraprasad** [comment]
@nekohacker591 , I am also facing this issue . If you got any solution kindly share it .

**Comment by henk717** [comment]
This is normal behavior and can be ignored
